modules/gridmap/editor/grid_map_editor_plugin.cpp

```diff
@@ -200,6 +200,7 @@ void GridMapEditor::_menu_option(int p_option) {
 }
 
 void GridMapEditor::_update_cursor_transform() {
+	ERR_FAIL_NULL(node);
```

Comment:
All callers of this either do the check or dereference `node` before, so this is also redundant.